wandb/cli/cli.py

```diff
@@ -1182,6 +1182,15 @@ def launch(
     elif resource is None:
         resource = "local-container"
 
+    host = api.settings("base_url")
```

Comment:
Is it possible that the api.settings doesn't pick up environment variables? in which case this should be `os.environ.get(WANDB_BASE_URL) or ...`